app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.UserProfileFragment">
+
+
+    <TextView
+        android:id="@+id/settings"
+        android:layout_width="126dp"
+        android:layout_height="47dp"
+        android:layout_marginTop="88dp"
+        android:text="@string/settings_title"
+        android:textColor="@color/seed"
+        android:textSize="30sp"
+        app:fontFamily="@font/pt_sans_bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.122"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/account_button"
+        android:layout_width="match_parent"
+        android:layout_height="55dp"
+        android:layout_marginLeft="35dp"
+        android:layout_marginTop="40dp"
+        android:layout_marginRight="35dp"
+        android:gravity="left|center_vertical"
+        android:text="@string/account_button"
+        android:textColor="@color/black"
```

Comment:
Custom theme colors should be used for automatic dark theme handling. To understand which color to use, check out the [theme builder](https://m3.material.io/theme-builder#/custom):